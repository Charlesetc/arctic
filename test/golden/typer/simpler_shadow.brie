define main [ : x [
  define x 2
  x
] ]
