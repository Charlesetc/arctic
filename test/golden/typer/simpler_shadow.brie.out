root[(ident(define):: ident(main):: block[][(block[ident(x)::][(ident(define):: ident(x):: ident(2)::)::, (ident(x)::)::]::Function)::Function]::Function)::]::